### PR TITLE
Update useSuccessToast to make it use AntD notifications; closes #2396.

### LINKED ISSUE #2396: Replace @blueprintjs Toaster with AntD Notifications
- see https://ant.design/components/notification/

- would be nice if existing toaster hooks API could be maintained to reduce refactoring required in other functions and components, but feel free to look at usage within existing locations like Apollo, and if it will be easier and faster to just strip it all out and start from scratch, then just do that and/or feel free to change accordingly if you see room for any improvement (ie: better functionality, etc)

- convert existing toasts as follows:
    - useSuccessToast() => notification.success()
    - useErrorToast() => notification.error()
    - useInfoToast() => notification.info()
    - useLoadingToast() => notification.open() - with loading-3-quarters icon
    - useDismissToast() => notification.close() - OR... look at the "Update Message Content" section and consider using that for changing notices in some scenarios, such as when entity loading is completed or a CRUD operation finishes, instead of closing the previous notice and opening a second one

- also add new
    - useWarningToast() => notification.warning()

- keep notifications in the bottom right corner by default, but allow that to be controlled via API

- you can likely keep the ToasterHash function for generating message IDs but feel free to change if you have an easier/better idea

### PR DESCRIPTION
**Pull request by nicolad**
## Problem this Pull Request solves

- Updates `useSuccessToast` item  from https://github.com/eventespresso/event-espresso-core/issues/2396

### CODE REVIEW

**Comment by nicolad** [comment]
Without `setTimeout` it was showing only the last notification:
Peek 2020-02-18 15-39

File: assets/src/application/services/toaster/useSuccessToast.tsx [hunk 1]
```diff
@@ -1,24 +1,22 @@
...
-						intent: Intent.SUCCESS,
-						icon: 'tick-circle',
-						message,
-						onDismiss,
-						timeout,
-					},
-					hash(message)
-				);
-			}
+	useCallback<any>(
```
Comment (manzoorwanijk) [inline assets/src/application/services/toaster/useSuccessToast.tsx]: why `<any>`?

File: assets/src/application/services/toaster/useToaster.tsx [hunk 2]
```diff
@@ -1,26 +1,17 @@
...
-		.reduce((a, b) => {
-			a = (a << 5) - a + b.charCodeAt(0);
-			return a & a;
-		}, 0)
-		.toString();
-};
-
 const useToaster = (): ToasterHook => {
 	const toaster = useContext(ToastContext);
+	const hash = uuidv4;
```
Comment (manzoorwanijk) [inline assets/src/application/services/toaster/useToaster.tsx]: `uuidv4` must be called to get the id - `uuidv4()`

Comment (nicolad) [reply assets/src/application/services/toaster/useToaster.tsx]: Updated https://github.com/eventespresso/event-espresso-core/pull/2408/commits/b08b78a3f0d9efa84e19ac3ee516a4db8432ee41

File: assets/src/application/services/toaster/useSuccessToast.tsx (hunk 1 above)
Comment (nicolad) [reply assets/src/application/services/toaster/useSuccessToast.tsx]: updated

**Pull request merged**